### PR TITLE
Update RN README

### PR DESCRIPTION
**Pull request by cbush**

### CODE REVIEW

**Review by mongomoe — APPROVED** [review]
LGTM 👍

File: xamarin/README.md [hunk 1]
```diff
@@ -1,4 +1,23 @@
...
+## Configuration
+
+Ensure realm-todo-dotnet/realm.json exists and contains the following properties:
+
+- **appId:** your Realm app ID, which can be found in the Realm app UI at https://realm.mongodb.com
+- **baseUrl:** the Realm backend URL. Should be https://realm.mongodb.com in most cases.
+
+## Run the app
+
+- Open realm-todo-app.sln in Visual Studio.
```
Comment (nlarew) [inline xamarin/README.md:12]:
```suggestion
- Open `realm-todo-app.sln` in Visual Studio.
```

File: react-native/README.md [hunk 2]
```diff
@@ -1,6 +1,11 @@
 # React Native Template App
+
 ## How to Run the Application for Mac Users:
-- `cd reactnative`
+- make sure you are in this directory
 - `npm install`
-- `cd ios` && `pod install --repo-update` && `cd ..`
+- `cd ios && pod install --repo-update cd ..`
```
Comment (nlarew) [inline react-native/README.md:6]:
```suggestion
- `cd ios && pod install --repo-update && cd ..`
```

File: react-native/README.md [hunk 3]
```diff
@@ -1,6 +1,11 @@
 # React Native Template App
+
 ## How to Run the Application for Mac Users:
-- `cd reactnative`
+- make sure you are in this directory
```
Comment (nlarew) [inline react-native/README.md:4]: Do they need to "ensure realm.json exists" like in .NET?

**Review by nlarew — APPROVED** [review]
A few changes then good to go

**Pull request merged**